drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-backend/src/test/java/org/drools/workbench/screens/guided/scorecard/backend/server/indexing/IndexGuidedScoreCardTest.java

```diff
@@ -37,6 +37,7 @@
 import org.uberfire.ext.metadata.io.KObjectUtil;
 import org.uberfire.java.nio.file.Path;
 
+@org.junit.Ignore
```

Comment:
Why is this now ```@Ignore```'ed?

Reply:
Oops :-( I accidentally committed this. It's fixed now.